config.ini

```diff
@@ -35,6 +35,19 @@ session.secure = False
 # wsgi server configuration
 ###
 
+# This configures Pyramid/Waitress to respect X-Forwarded-* headers which we
```

Comment:
It's because when pypicloud runs behind an ALB, it's serving over HTTP even though the user is accessing it over HTTPS -- so when pypicloud builds an absolute link, which it seems to do to load static assets, it always uses http:// unless you tell it to respect X-Forwarded-Proto.